app/assets/stylesheets/app/_sizes.scss

```diff
@@ -0,0 +1,10 @@
+// TEXT SIZE
+// $size-1: 3.5rem;
+// $size-2: 3rem;
+// $size-3: 2.5rem;
+// $size-4: 2rem;
+// $size-5: 1.5rem;
+// $size-6: 1rem;
```

Comment:
are we gonna need them later?

Reply:
Bulma defines and uses the `$size-1` to `$size-7`. The commented values are the default values kept only for reference, since they match our specs.